nullaway/src/test/java/com/uber/nullaway/NullAwayTypeUseAnnotationsTests.java

```diff
@@ -38,16 +38,28 @@ public void annotationAppliedToTypeParameter() {
             "import java.util.List;",
             "import java.util.ArrayList;",
             "import org.checkerframework.checker.nullness.qual.Nullable;",
+            "import org.checkerframework.checker.nullness.qual.NonNull;",
             "class TypeArgumentAnnotation {",
             "  List<@Nullable String> fSafe = new ArrayList<>();",
             "  @Nullable List<String> fUnsafe = new ArrayList<>();",
             "  void useParamSafe(List<@Nullable String> list) {",
             "    list.hashCode();",
             "  }",
+            "  void unsafeCall() {",
+            "    // BUG: Diagnostic contains: passing @Nullable parameter",
+            "    useParamSafe(null);",
```

Comment:
Before this change, this was a false negative? (Either way, great to have the test, but just curious)

Reply:
Yes that's right, this was a false negative before